src/main/java/uk/gov/hmcts/reform/sendletter/model/out/LetterStatus.java

```diff
@@ -26,13 +28,15 @@
 
     public LetterStatus(
         final UUID id,
+        final String status,
         final String messageId,
         final ZonedDateTime createdAt,
         final ZonedDateTime sentToPrintAt,
         final ZonedDateTime printedAt,
         final boolean hasFailed
```

Comment:
can we have a todo here - i think `hasFailed` is redundant now and can be removed in next pr

Reply:
Though if it is part of our public API that would be a breaking change.